index-test.js

```diff
@@ -1,20 +1,59 @@
-/*
+// MINIMUM requirement of Unit Tests (as stated on project brief)
 
-MINIMUM requirement of Unit Tests (as stated on project brief)
+//test("Submitting a new task adds it to the list", () => {
+// test goes here
+//});
 
-test("Submitting a new task adds it to the list", () => {
-  // test goes here
-});
+setTimeout(() => {
 
-test("Checking an entry marks it as complete", () => {
-  // test goes here
-});
+  test("Checking first item marks it as complete", () => {
```

Comment:
Thank You for completing the checkbox functioning tests, YAAAAY there amazing 🎉 🥳